eclipse-ee4j/jaxrs-api
# TCK Migration: Move pending jaxrs/ee/rs/core,client,ext tests from jakartaee-tck

This is to migrate the pending tests from jaxrs/ee/rs/core,client,ext folders in jakartaee-tck

To test the changes :

export JAVA_HOME=<JDK11_HOME>
export M2_HOME=<MAVEN_HOME>
export PATH=$M2_HOME/bin:$JAVA_HOME/bin:$PATH

jaxrs-tck/
mvn clean install

jersey-tck/
run all tests : mvn clean verify -Parq-glassfish-managed

The tests migrated : 

- jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/rs/core (jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/rs/core/request covered in https://github.com/eclipse-ee4j/jaxrs-api/pull/1002)
- jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/rs/client (jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/rs/client/asyncinvoker covered in https://github.com/eclipse-ee4j/jaxrs-api/pull/1002)
- jakartaee-tck/src/com/sun/ts/tests/jaxrs/ee/rs/ext
